flake.nix

```diff
@@ -177,18 +177,52 @@
             rust-bin.stable.latest.minimal
             valgrind
           ];
+
+          dockerHoprdEntrypoint = pkgs.writeShellScriptBin "docker-entrypoint.sh" ''
+            set -euo pipefail
+
+            # if the default listen host has not been set by the user,
+            # we will set it to the container's ip address
+
+            listen_host="''${HOPRD_DEFAULT_SESSION_LISTEN_HOST:-}"
```

Comment:
Would be nice to document these env vars in the main README.md

Reply:
But these are not application specific. There only apply to the docker setup.

Reply:
Its a cfg value, thus already documented as part of the config file docs.